node/pegnet/grading.go

```diff
@@ -110,7 +110,16 @@ func (p *Pegnet) InsertRates(tx *sql.Tx, height uint32, rates []opr.AssetUint, p
 			totalCapitalization.Add(totalCapitalization, assetCapitalization)
 		}
 		if issuance[fat2.PTickerPEG] == 0 {
-			ratePEG.Set(totalCapitalization)
+			// This number is outrageously large since the total cap is the total
```

Comment:
This comment is about the old value. I would change the comment to something like
```go
// If there are no PEGs in the system, PEGs have no value (divide-by-zero)
// At least one block will have to be mined in order for PEGs to attain a value
```

and post the reasoning into the PR itself

Reply:
That's fair. I'll change it